### DISCUSSION TITLE
Clip skip setting with XL radio button

### DISCUSSION THREAD

**Discussion by djrawlins** [lllyasviel/stable-diffusion-webui-forge#2292] [Q&A]
If I use the XL radio button I can’t then see the clip skip dropdown (installed in quick settings). Is there a way to change the clip skip setting while in the XL "tab"? I use it a lot for Loras along with the Diffusion bits in low bits dropdown. I can’t find a way to set the clip skip otherwise as I don’t see it in the settings section.

Clip skip is there on the All radio button but that seems to defeat the object of having the presets. It also resets the other XL settings I may have made so it takes longer to reinstate them.
Thanks

**Comment by ride5k** [comment]
came here for this.  behavior is counterintuitive, cannot find a way to expose clip skip settings to UI either in txt2 or img2 or main quicksettings for "modes" other than "all."
